test/lib/cli/output_test.exs

```diff
@@ -0,0 +1,172 @@
+# credo:disable-for-this-file Credo.Check.Readability.RedundantBlankLines
+defmodule Xcribe.CLI.OutputTest do
+  use ExUnit.Case, async: true
+
+  alias Xcribe.CLI.Output
+  alias Xcribe.DocException
+  alias Xcribe.Request.Error
+
+  import ExUnit.CaptureIO
+
+  describe "print_request_errors/1" do
+    test "parsing errors" do
+      # |> document(as: "some cool description")
+
+      route_error = %Error{
+        type: :parsing,
+        message: "route not found",
+        __meta__: %{
+          call: %{
+            description: "test name",
+            file: File.cwd!() <> "/test/lib/cli/output_test.exs",
+            line: 13
+          }
+        }
+      }
+
+      # |> document(as: "awesome route")
```

Comment:
:eyes:

Reply:
I need a line with to test the output. Test if we are able to point the document macro.

Reply:
Ah, I see... maybe something to make this clearer? :thinking: 

And what about line 13? What is that comment doing?

Reply:
Do you mean a comment to explain the comments? 

(the line 13 has the same purpose)